apps/runner/cmd/runner/config/config.go

```diff
@@ -33,6 +33,7 @@ type Config struct {
 	ResourceLimitsDisabled bool   `envconfig:"RESOURCE_LIMITS_DISABLED"`
 	DaemonStartTimeoutSec  int    `envconfig:"DAEMON_START_TIMEOUT_SEC"`
 	SandboxStartTimeoutSec int    `envconfig:"SANDBOX_START_TIMEOUT_SEC"`
+	UseDaemonEntrypoint    bool   `envconfig:"USE_DAEMON_ENTRYPOINT"`
```

Comment:
I suggest we inverse the logic here.

`USE_CONTAINER_ENTRYPOINT` or something like that.
I think Daemon as entrypoint should be the preferred way to run.